### DISCUSSION TITLE
Overlay avoid closing clicking outside

### DISCUSSION THREAD

**Discussion by manlioesp** [unpoly/unpoly#264] [Q&A]
Hi to everybody! I'm new to unpoly. I have added in a new layer a script for a datepicker using up.compiler.
Everything is working but as soon as I click on the date picker the overlay layer is closed.
How can I configure unpoly to avoid closing the modal when I click on the background?
Thank you

**Comment by adam12** [comment]
Hello!

You likely want to tweak the dismissible setting for that layer. See https://unpoly.com/a-up-layer-new#up-dismissable

**Reply by manlioesp** [reply]
Thank you very much, it works like a charm. 
I missed that part of the page, have a nice day.
